config/routes.rb

```diff
@@ -28,6 +29,14 @@
       param: :slug,
       constraints: {slug: %r{[^/]+/[^/]+}}
 
-    get 'search', controller: 'v2/search', action: 'search'
+    get '/:slug',
+      controller: 'v2/organizational_units',
```

Comment:
It's related to this line, since this is the route for `organizational_units`. To integrate the frontent better, another route `/organizational_units/:slug` would be nice. It should do the same thing as the route above.